atef/widgets/config/page.py

```diff
@@ -2207,14 +2100,41 @@ def select_comparison_type(self, new_type_index: int) -> None:
                 self.specific_combo.setCurrentText(type_name)
                 return
         comparison = cast_dataclass(data=self.data, new_type=new_type)
-        self.parent_tree_item.widget.replace_comparison(
-            old_comparison=self.data,
-            new_comparison=comparison,
-            comp_item=self.tree_item,
-        )
-        self.tree_item.setText(1, new_type.__name__)
-        self.new_comparison(comparison=comparison)
-        self.update_context()
+        parent_widget = self.full_tree.maybe_get_widget(self.parent_tree_item)
+
+        with self.full_tree.modifies_tree(select_prev=False):
+            # TODO: Still need to replace in dataclass.
```

Comment:
TODO reminder (how important is this to end user?)

Reply:
Also fixed 🤦